libs/util/shared/src/lib/models/search.model.ts

```diff
@@ -20,6 +20,11 @@ export interface Organisation {
   recordCount?: number
 }
 
+export interface OrganisationApiModel {
+  key: string
+  doc_count: number
+}
```

Comment:
This can probably stay in the scope of the organisations service? it might not be useful to anyone else